drivers/modem/gsm_ppp.c

```diff
@@ -44,6 +47,10 @@ LOG_MODULE_REGISTER(modem_gsm, CONFIG_MODEM_LOG_LEVEL);
 #define GSM_RSSI_RETRIES                10
 #define GSM_RSSI_INVALID                -1000
 
+#if defined(CONFIG_MODEM_GSM_ENABLE_GNSS)
```

Comment:
Why use two different styles? On line 25 you use `#ifdef CONFIG_MODEM_GSM_ENABLE_GNSS`

Reply:
Changed style to #if defined()